### DISCUSSION TITLE
How to update the DWIN display firmware

### DISCUSSION THREAD

**Discussion by JoeMaxey** [mriscoc/Ender3V2S1#21] [Q&A]
First, thanks to Miguel for this great firmware. Installed yesterday and am loving it so far. Great work!

I'm trying to update my display to the icons, but am having trouble finding the DWIN_SET directory. I've found the instructions for this update, but am unable to find the DWIN folder that I'm supposed to copy to my uSD card. 

Thanks!

**Comment by mriscoc** [comment]
What Icon set do you want? https://github.com/mriscoc/Ender3V2S1/wiki/How-to-update-the-display

**Reply by JoeMaxey** [reply]
Thanks, Miguel. I didn't realize the title to the bitmap library was a link to the DWIN folder. My bad!

Do I copy all the files plus the DWIN folder to my card when flashing my display or just the DWIN folder?

**Reply by mriscoc** [reply]
The DWIN_SET folder.
![image](https://user-images.githubusercontent.com/2745567/152649824-c606c0cc-6294-423f-b1a7-24cb6da1b63e.png)

**Reply by JoeMaxey** [reply]
Worked perfectly. Thanks so much, Miguel!

**Reply by rshutterbug** [reply]
Hi, Newbie here, so please be gentle! 

Ender 3V2 with a 4.2.2 board and a DWIN display board. I flashed your firmware without problem, so thought it would be nice to update the display. When powering on the printer with the display SD card inserted, it first went blue and after a few seconds orange. I removed the SD card and rebooted, but all I get is a black display. 

The printer was still "live" at this point because pressing the knob gave a beep. Tried reflashing the display several times, no further display, just black. I then thought I'd try reflashing your printer firmware, as there were a few threads that said this had worked. After reflashing the printer, I've still got a black display and now no beeps if I press the knob.

Any help or suggestions gratefully received, or failing all else how do I get back to the factory firmwares to start again!
